sled-agent/src/services.rs

```diff
@@ -2932,11 +2929,33 @@ impl ServiceManager {
             //
             // This is (currently) intentional, as the zone filesystem should
             // be destroyed between reboots.
-            let mut rng = rand::thread_rng();
```

Comment:
Should probably remove the above comment and/or move part of it to the else clause below.

Reply:
Done in 61d94dce573b33d019dfe41023e4685285e410e1